IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml

```diff
@@ -2949,26 +2950,21 @@ components:
           $ref: '#/components/schemas/CurrencyDef'
           description:
             The currency returned might be different from the one requested.
-        
 
     TripOfferRequest:
       type: object
       properties:
-        tripId:
-          type: string
-          format: uuid
-          example: d290f1ee-6c54-4b01-90e6-d701748f0851
-          description: >-
-            optionally used instead of tripSearchCriteria element, if the offers
-            are to be constructed for a previously built trip
-        tripRef:
-          type: string
-          format: url
-          example: 'http://some.provider/trip/2345'
-        trip:
-          $ref: '#/components/schemas/Trip'
-        requestedSection:
-          $ref: '#/components/schemas/TravelSectionDef'
+        tripSearch:
+          oneOf:
```

Comment:
we should clear the oneOff construct. As the generated code is too bad we decided to have the oneOff only in cmments and not as part of the schema

Reply:
I understand the point and can agree but IMO this is something to be discussed with the group so it is clear to everyone, and then to be applied to the whole specifications (not a single oneOf structure should then remain)